src/custom-types.json

```diff
@@ -78,7 +78,11 @@
   },
   "application/javascript": {
     "charset": "UTF-8",
-    "compressible": true
+    "compressible": true,
+    "extensions": ["mjs"],
+    "sources": [
+      "https://github.com/nodejs/node-eps/blob/master/002-es-modules.md"
```

Comment:
It links to https://github.com/tc39/ecma262/issues/322 which rejects the new subtype of `+module` and can be cross referenced in https://github.com/whatwg/html/issues/558 and step 7 of https://html.spec.whatwg.org/multipage/webappapis.html#fetch-a-single-module-script . I will write up a PR and merge it for the node EP explicitly stating that it needs to be the same.